internal/controller/progressive/progressive_test.go

```diff
@@ -107,39 +111,17 @@ func Test_processUpgradingChild(t *testing.T) {
 		expectedError             error
 	}{
 		{
-			name:                      "no upgrading child status on the live rollout",
-			rolloutObject:             defaultMonoVertexRollout.DeepCopy(),
-			existingUpgradingChildDef: createMonoVertex("test"),
-			expectedDone:              false,
-			expectedRequeueDelay:      assessmentSchedule.Interval,
-			expectedError:             nil,
-		},
-		{
-			name: "preset upgrading child status on the live rollout - different name",
-			rolloutObject: setMonoVertexProgressiveStatus(
-				defaultMonoVertexRollout.DeepCopy(),
-				&apiv1.UpgradingMonoVertexStatus{
-					UpgradingPipelineTypeStatus: apiv1.UpgradingPipelineTypeStatus{
-						UpgradingChildStatus: apiv1.UpgradingChildStatus{Name: "test"}},
-				},
-				nil),
-			//setRolloutObjectChildStatus(defaultMonoVertexRollout.DeepCopy(), &apiv1.UpgradingChildStatus{Name: "test"}, &apiv1.PromotedChildStatus{}),
-			existingUpgradingChildDef: createMonoVertex("test-1"),
-			expectedDone:              false,
-			expectedRequeueDelay:      assessmentSchedule.Interval,
-			expectedError:             nil,
-		},
```

Comment:
these test cases no longer make sense - this function now assumes that the UpgradingChildStatus is the correct one